templates/vue/src/components/NodeModal.vue

```diff
@@ -470,44 +510,37 @@ export default {
         }
       }
     },
-    async handleSubmit() {
+    handleSubmit() {
       this.loading = true
       this.formErrors = this.validateNode()
       if (!this.formErrors.length) {
-        this.node.status = "publish"
         this.updateNodeCoordinates()
-        this.loading = true
 
         if (this.node.mediaType === "url-embed" && this.node.behaviour !== "embed") {
-          await this.setLinkData()
+          ;(async () => await this.setLinkData())()
         }
 
         if (this.shouldReloadDuration()) {
           this.loadDuration = true
         } else {
-          return this.submitNode()
+          return (async () => await this.submitNode())()
```

Comment:
@wynnset Any reason why this was changed to not use async await? I think it's cleaner the way it was before.

Reply:
I changed this because the `loading` data property was immediately changing to false and so the node modal was not disabling the buttons while that was happening and I could click cancel or submit again while the node was submitting. Do you have a suggestion of how to avoid that in a different way?